src/ff7_opengl.cpp

```diff
@@ -318,6 +318,9 @@ void ff7_init_hooks(struct game_obj *_game_object)
 	//######################
 	if (enable_time_cycle)
 	{
+		replace_call_function(ff7_externals.battle_draw_text_ui_graphics_objects_call, ff7::battle::draw_ui_graphics_objects_wrapper);
+		replace_call_function(ff7_externals.battle_draw_box_ui_graphics_objects_call, ff7::battle::draw_ui_graphics_objects_wrapper);
+
 		replace_call_function(ff7_externals.world_wm0_overworld_draw_all_74C179 + 0x175, ff7::world::wm0_draw_minimap_quad_graphics_object);
         replace_call_function(ff7_externals.world_wm0_overworld_draw_all_74C179 + 0x1BE, ff7::world::wm0_draw_world_effects_1_graphics_object);
         replace_call_function(ff7_externals.world_wm0_overworld_draw_all_74C179 + 0x208, ff7::world::wm0_draw_minimap_points_graphics_object);
```

Comment:
Can you please indent these lines too?

Reply:
These look already indented to me, isn't this just github showing a line in two?

Reply:
They seems to be aligned on the left compared to the previous `replace_call_function` functions. Probably a missing tab or spaces.

Reply:
Should be good now (the ones on the bottom were using spaces so I changed them to tabs)

Reply:
Fantastic, thank you :)